src/Modules/SimplCommerce.Module.Catalog/Views/Brand/BrandDetail.cshtml

```diff
@@ -1,130 +1,138 @@
+											@category.Name
+											<small>(@category.Count)</small>
+										</label>
+									</li>
+								}
+							</ul>
+						</div>
+					</div>
+				</div>
+			</div>
+			@if (Model.FilterOption.Price.MaxPrice != Model.FilterOption.Price.MinPrice)
+			{
+				<div id="accordion-price" class="mt-4">
+					<div class="card">
+						<div class="card-header" id="cardHeaderPrice">
+							<h5 class="mb-0">
+								<a data-toggle="collapse" href="#collapse-price" aria-expanded="true" aria-controls="collapse-price">
+									@Localizer["Price"]
+									<i class="fa fa-angle-down"></i>
+								</a>
+							</h5>
+						</div>
+						<div id="collapse-price" class="collapse show" aria-labelledby="cardHeaderPrice" data-parent="#accordion-price">
+							<div class="card-body">
+								<div id="priceSlider"></div>
+								<div>
+									<input type="hidden" id="originMinPrice" value="@Model.FilterOption.Price.MinPrice.ToString("F0")" />
+									<input type="hidden" id="originMaxPrice" value="@Model.FilterOption.Price.MaxPrice.ToString("F0")" />
+									<span id="minPrice"></span>
+									<span id="maxPrice"></span>
+								</div>
+							</div>
+						</div>
+					</div>
+				</div>
+			}
+		</form>
+	</div>
+	<div class="col-md-9">
+		<div class="row mb-3">
+			<div class="col-sm-6 category-result">
+				<h2>@Model.BrandName</h2>
+				<small class="badge-results">@Model.TotalProduct @Localizer["results"]</small>
+			</div>
+			<div class="col-sm-6">
+				<ul class="list-inline clear-both product-list-display-options">
+					<li class="sort-by">
+						<label>@Localizer["Sort by:"]</label>
+						<div class="btn-group">
+							<select asp-for="CurrentSearchOption.Sort" asp-items="Model.AvailableSortOptions" class="form-control"></select>
+						</div>
+					</li>
+					@*<li class="pagination-option">
+						    <label>Show :</label>
+						    <div class="btn-group">
+							   <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown">
+								  20 <span class="caret"></span>
+							   </button>
+							   <ul class="dropdown-menu" role="menu">
+								  <li><a href="#">All</a></li>
+								  <li><a href="#">50</a></li>
+								  <li><a href="#">30</a></li>
+							   </ul>
+						    </div>
+						</li>*@
+				</ul>
+			</div>
+		</div>
+		<div class="row product-list">
+			@foreach (var product in Model.Products)
+			{
+				<div class="col-xs-6 col-md-4">
+					<partial name="_ProductThumbnail" model="product" />
+				</div>
+			}
+		</div>
+		<div class="row">
+			<div class="col-md-12 text-right">
+				<cs-pager cs-paging-pagesize="@Model.CurrentSearchOption.PageSize"
+						cs-paging-pagenumber="@Model.CurrentSearchOption.Page"
+						cs-paging-totalitems="@Model.TotalProduct"
+						cs-pagenumber-param="page"
+						asp-basehref="~/@Model.BrandSlug"
+						asp-all-route-data="@Model.CurrentSearchOption.ToDictionary()"></cs-pager>
+			</div>
+		</div>
+	</div>
 </div>
```

Comment:
I did it, its still not working

Reply:
I did it, its working. From Tools->Options->Text Editor->Html->Tabs. I made tab size and indent size 4. Indebting Smart and it inserts spaces instead of tabs. Is there anything else I can do to resolve this?